src/GraphQL/Instrumentation/ApolloTrace.cs

```diff
@@ -3,51 +3,113 @@
 
 namespace GraphQL.Instrumentation
 {
+    /// <summary>
+    /// Contains Apollo tracing metrics.
+    /// </summary>
     public class ApolloTrace
     {
+        /// <summary>
+        /// Initializes a new instance with the specified parameters.
+        /// </summary>
+        /// <param name="start">The UTC date and time that the GraphQL document began execution.</param>
```

Comment:
We have no control over the `DateTime.Kind` here. It may be any kind.

Reply:
Discussed below.